avm/ptn/app-service-lza/hosting-environment/modules/supporting-services/modules/key-vault.bicep

```
@@ -0,0 +1,151 @@
+targetScope = 'resourceGroup'
+
+// ------------------
+//    PARAMETERS
+// ------------------
+
+@description('The location where the resources will be created.')
+param location string = resourceGroup().location
+
+@description('The name of the Key Vault.')
+param keyVaultName string
+
+@description('Optional. The tags to be assigned to the created resources.')
+param tags object = {}
+
+@description('Required. Whether to enable deplotment telemetry.')
+param enableTelemetry bool
+
+@description('The resource ID of the Hub Virtual Network.')
+param hubVNetId string
```

Same comment as with the other resource IDs

```suggestion
param hubVNetResourceId string
```